vita3k/modules/SceProcessmgr/SceProcessmgr.cpp

```diff
@@ -219,7 +242,12 @@ EXPORT(int, sceKernelLibcMktime, struct tm *date, VitaTime *time, uint64_t *para
         date->tm_year -= 1900;
         year_1900 = true;
     }
-    auto time_local = mktime(date);
+    tm host_tm;
+    // Copy the input date to host_tm and use that on mktime instead of the input directly
+    // to avoid stack corruption on systems where tm size is different
+    memcpy(&host_tm, &date, sizeof(VitaTM));
```

Comment:
This line is not needed.

Reply:
Are you sure? seems that we does not send any data to host mktime

Reply:
Indeed you are right, my bad.

Reply:
pushed the line back